Public/Src/Engine/UnitTests/Scheduler/PipFingerprinterTests.cs

```diff
@@ -380,6 +380,100 @@ public void TestUserProfileStaysOutOfFingerprints()
             XAssert.IsTrue(json.Contains(Path.GetFileName(Environment.GetFolderPath(Environment.SpecialFolder.InternetCache)).ToUpperInvariant()));
         }
 
+        [Fact]
+        public void TestUntrackedPathStaysOutOfFingerprints()

```

Comment:
This name is misleading, because untracked paths still remain part of the fingerprint. 
```suggestion
        public void TestUntrackedExecutablePathStaysOutOfFingerprints()
```